### DISCUSSION TITLE
how to disable merge in MergeTree engine

### DISCUSSION THREAD

**Discussion by davidkeeper1983** [ClickHouse/ClickHouse#42790] [Q&A] [answered]
Background: we use clickhouse as a TSDB scenario, so we don't need update/delete, we only append a lot of datas into clickhouse, and we hope clickhouse can use all IO write that can reach limit of disk IO write.
From now on we can only see a command 'SYSTEM STOP MERGES [[db.]merge_tree_family_table_name]' to stop merge manually. can Clickhouse have a system parameter to disable the merge in background?
thanks ~~

**Comment by UnamedRus** [comment]
Merges are not part of update/delete workflow.
No need to disable them.

**Comment by alexey-milovidov** [comment, marked as answer]
Merges are required to keep the system operational.
There is no way to disable merges and keep the system working.

**Reply by davidkeeper1983** [reply]
Thanks a lot, but still I want to clarifying that our system want to write more and more datas than read data, so we hope clickhouse can stop merge(will taken a lot o read IO can cause write slow down to 1/3 of max speed.)

**Reply by davidkeeper1983** [reply]
Another way: Can clickhouse merge in memory (like sst table) before write to disk.

**Reply by alexey-milovidov** [reply]
No, it's also not possible.
